app/main.py

```diff
@@ -1,2 +1,49 @@
+class Node:
+    def __init__(self, key: object, value: object) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 16, load_factor: float = 0.75) -> None:
+        self.capacity = capacity
+        self.size = 0
+        self.load_factor = load_factor
+        self.table = [[] for _ in range(capacity)]
```

Comment:
Better fill out initial table with None values - no need to create 1 million empty lists. Also annotate self.table that it can be a list of Nodes or None values